gatewayd.yaml

```diff
@@ -3,7 +3,7 @@
 loggers:
   default:
     output: ["console"] # "stdout", "stderr", "syslog", "rsyslog" and "file"
-    level: "info" # panic, fatal, error, warn, info (default), debug, trace
+    level: "debug" # panic, fatal, error, warn, info (default), debug, trace
```

Comment:
Looks like a forgotten debug

Reply:
yup, thanks for looking this over!
the PR is not ready yet. I just changed this YAML to show it as an example. Probably in the final version of the PR, I will change the YAML to the default mode with commented proxy definitions.